examples/with-reason-relay/next.config.js

```diff
@@ -1,21 +0,0 @@
-require('dotenv').config()
-const path = require('path')
-const Dotenv = require('dotenv-webpack')
-
-module.exports = {
-  webpack: (config) => {
-    config.plugins = config.plugins || []
-
-    config.plugins = [
-      ...config.plugins,
-
-      // Read the .env file
-      new Dotenv({
```

Comment:
Is the environment variable that was previously injected used client side? If so we will need to rename it to `NEXT_PUBLIC_` so it still is

Reply:
Good catch!